other-cel/block-pod-exec-by-namespace/block-pod-exec-by-namespace.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: deny-exec-by-namespace-name
+  annotations:
+    policies.kyverno.io/title: Block Pod Exec by Namespace Name in CEL expressions
+    policies.kyverno.io/category: Sample in CEL 
+    policies.kyverno.io/minversion: 1.11.0
+    kyverno.io/kubernetes-version: "1.26-1.27"
+    policies.kyverno.io/subject: Pod
+    policies.kyverno.io/description: >-
+      The `exec` command may be used to gain shell access, or run other commands, in a Pod's container. While this can
+      be useful for troubleshooting purposes, it could represent an attack vector and is discouraged.
+      This policy blocks Pod exec commands to Pods in a Namespace called `pci`.
+spec:
+  validationFailureAction: Enforce
+  background: false
+  rules:
+  - name: deny-exec-ns-pci
+    match:
+      any:
+      - resources:
+          kinds:
+          - Pod/exec
+          operations:
+          - CONNECT
+    validate:
+      cel:
+        expressions:
+          - expression: "request.namespace != 'pci'"
```

Comment:
Can we use the `namespaceObject` instead?
```
expression: "namespaceObject.metadata.name != 'pci'"
```

Reply:
Sure. I've made the changes.